scripts/all.fish

```diff
@@ -0,0 +1,11 @@
+for features in "--all-features" "--features runtime-benchmarks" "--features try-runtime"
```

Comment:
Would it also make sense to test with no (default) features, and with the mock feature, where specified? Anyway very good solution to the problem!

Reply:
yes that's a good idea

Reply:
default features are a good idea. But mock doesn'T work since not all of the crates have that feature